modules/react/layout/stories/Grid.stories.mdx

```diff
@@ -30,66 +25,66 @@ yarn add @workday/canvas-kit-react
 
 ### Basic Example
 
-Below is a basic layout built with `Grid`. We created four components: `Header`, `SideBar`,
-`BodyContent` and `Footer`. We also used those names for their respective `gridArea` props. Then we
-can use those names in the parent component, `UIExample`, to organize the UI however we want. This
-allows the parent container to control how its children render. For page layouts, Workday typically
-uses a 12 column grid. In the example below, the Sidebar is 3 columns and the Body Content is 9
-columns.
+> **Note**: We recommend you familiarize yourself with CSS Grid
+> ([MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids),
+> [CSS-Tricks](https://css-tricks.com/snippets/css/complete-guide-grid/)) before diving into our
+> `Grid` component. This example makes use of
```

Comment:
I updated this paragraph to reference the CSS-Tricks guide to grid in addition to the MDN article on grid.